source/backend/dal/Repositories/EntityNoteRepository.cs

```diff
@@ -100,6 +109,24 @@ public bool DeleteAcquisitionFileNotes(long entityId)
             return false;
         }
 
+        public bool DeleteLeaseFileNotes(long noteId)
+        {
+            var leaseFileNotes = this.Context.PimsLeaseNotes.
+                                        Include(ln => ln.Note).
+                                        Where(x => x.NoteId == noteId).ToList();
+
+            if (leaseFileNotes.Any())
+            {
+                foreach (var leaseFileNote in leaseFileNotes)
+                {
+                    this.Context.PimsLeaseNotes.Remove(leaseFileNote);
```

Comment:
I wonder if we need this now or if our cascades or covering this. Maybe we need to go back now that we have some cascade logic and clean some of these up? certainly not something you need to worry about this PR but I should probably log tech debt for it.